mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java

```diff
@@ -488,15 +488,15 @@ public void processConnection() throws Exception {
           openClientSockets.add(socket);
 
           if (protocolNegotiationEnabled) {
-            Platform.get().configureTlsExtensions(sslSocket, null, protocols);
+            Platform.Companion.get().configureTlsExtensions(sslSocket, null, protocols);
```

Comment:
Do we want to use `@JvmStatic` to minimize these changes in unrelated files?

Reply:
Yep, good idea.  I'm going through the set of steps in the description, should have marked as WIP.  Thanks for the early review.